src/Project/Generics.jl

```diff
@@ -0,0 +1,176 @@
+#=
+ MIT License
+
+ Copyright (c) 2010-present David A. Kopriva and other contributors: AUTHORS.md
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ --- End License
+=#
+
+#
+# Creating curves
+#
+"""
+    new(name::String,
+        xEqn::String,
+        yEqn::String,
+        zEqn::String = "z(t) = 0.0" )
+
+Create a new parametric equation curve.
+"""
+function new(name::String,
```

Comment:
As already discussed:
```suggestion
function create(name::String,
```
or essentially anything but `new` ;-)

Reply:
And already answered.

Reply:
As I've mentioned above (I don't know how to link here), I'm not a fan of a single function name, but prefer a descriptive noun+verb naming system. This makes it way easier to read code later on, too. I like self documenting code. The short "generic" versions are options for interactive use by those who get tired of typing. They should be considered "advanced" techniques.